_tools/libddwaf-updater/update.sh

```diff
@@ -61,6 +62,7 @@ echo Updating libddwaf for darwin/amd64yes
 curl -L https://github.com/DataDog/libddwaf/releases/download/$version/libddwaf-$version-darwin-x86_64.tar.gz | tar -xz -C$tmpdir
 echo Copying the darwin/amd64 library
 cp -v $tmpdir/libddwaf-$version-darwin-x86_64/lib/libddwaf.a.stripped $bindings_dir/lib/darwin-amd64/libddwaf.a
+cp -v "$tmpdir/libddwaf-$version-darwin-x86_64/lib/libddwaf.dylib" "$bindings_dir/purego/embed/darwin-amd64"
```

Comment:
1. Do you mind keeping the file extension please?
2.  WDYT of putting them along with the static libs instead of a new folder?
3. The destination dir is still the old  `purego/embed/` one
Same comment for the other files please 🙏

Reply:
done 👍